image/autodetect.go

```diff
@@ -103,8 +102,8 @@ func Autodetect(path string) (string, error) {
 	case header.MediaType == string(schema.MediaTypeManifestList):
 		return TypeManifestList, nil
 
-	case header.MediaType == "" && header.SchemaVersion == 0 && header.Config != nil:
-		// config files don't have mediaType/schemaVersion header
+	case header.RootFS != nil:
```

Comment:
I think only need a unique attribute can determine the Config type, the other on some superfluous